include/text.php

```diff
@@ -41,18 +41,28 @@ function replace_macros($s, $r) {
 	return $output;
 }
 
+// PHP < 7 polyfill
+if (!is_callable('intdiv')) {
+	function intdiv($a, $b) {
+		return ($a - $a % $b) / $b;
+	}
+}
 
-// random string, there are 86 characters max in text mode, 128 for hex
-// output is urlsafe
+/**
+ * @brief Generates a pseudo-random string of hexadecimal characters
+ *
+ * Only supports pair numbers of output characters.
+ *
+ * @param int $size
+ * @return string
+ */
+function random_string($size = 64)
```

Comment:
Was the second parameter never used?

Reply:
Nope, the other mode generated random base64 encoded characters, which was unwieldy since not all produced characters after decoding would be printable.